myuw/models/__init__.py

```diff
@@ -368,3 +368,19 @@ def __str__(self):
     class Meta(object):
         app_label = 'myuw'
         db_table = 'migration_preference'
+
+
+class ResourceCategoryPin(models.Model):
+    resource_category_id = models.CharField(max_length=255)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    @staticmethod
+    def get_user_pinned_categories(user):
+        try:
```

Comment:
To increase test coverage, you may want to change to:
if ResourceCategoryPin.objects.filter(user=user).exists():
   ...